README.md

```diff
@@ -133,24 +132,24 @@ Alternatively disable signing in each target:
 * To enable test schemes: `./scripts/setup_spm_tests.sh`
 * `open Package.swift` or double click `Package.swift` in Finder.
 * Xcode will open the project
-  * Choose a scheme for a library to build or test suite to run
```

Comment:
These spaces are required for the markdown generation. Please restore. See the markdown generation.